src/lib/components/PrismicEmbed.svelte

```diff
@@ -1,9 +1,14 @@
-<!-- TODO: Why are embeds 200px wide? -->
-
 <script lang="ts">
+  import * as prismicH from "@prismicio/helpers"
   import type * as prismicT from "@prismicio/types"
 
   export let field:prismicT.EmbedField
+
+  if(!prismicH.isFilled.embed(field)) {
+    console.warn("Invalid prop")
+  }
 </script>
 
-{@html field.html}
+{#if prismicH.isFilled.embed(field)}
+  {@html field.html}
+{/if}
```

Comment:
It looks like not all files have been formatted with Prettier. Could you install the Svelte Prettier plugin and run the global format command?

```sh
npm install --save-dev prettier-plugin-svelte
npm run format
```

Reply:
👍